topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -61,7 +61,7 @@ def load_metadata(self, asset: Asset = None, metadata_file: str = "", is_load_al
             asset.key_name = AssetKey.Visual
             self.populate_item(asset_metadata, asset)
 
-    def populate_item(self, metadata_row, asset: Asset = None) -> None:
+    def populate_item(self, metadata_row: Dict[str, str], asset: Asset = None) -> None:
```

Comment:
Is `metadata_row` always a `Dict[str, str]`? Can it be a Dict[str, int]` or `Dict[str, float]` too?

Reply:
Just noting we took a look and it is Dict[str, str] but good to check it.